tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -31,13 +32,15 @@
 
 C_SPEED_OF_LIGHT = const.c.to("cm/s").value
 
-from tardis.io.logger.montecarlo_tracking import log_decorator
-from tardis.io.logger import montecarlo_tracking as mc_tracker
 
-# @log_decorator
 @njit
 def single_packet_loop(
-    r_packet, numba_model, numba_plasma, estimators, vpacket_collection
+    r_packet,
+    numba_model,
+    numba_plasma,
+    estimators,
+    vpacket_collection,
+    r_packet_track,
```

Comment:
Is there a better name for this? It's a bit confusing to have `track_r_packet` and `r_packet_track`

Reply:
Yes, please do let me know your opinions on a new name for the variable. It should be concise & should be easily called via the `sim` simulation object.

Reply:
Renamed to `tracked_rpacket_properties` 😄